db/migrate/20230301161010_add_likes_counter_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddLikesCounterToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :likes_counter, :integer
+  end
+end
```

Comment:
- [ ] You did an excellent job of incorporating the posts count into the user table and the likes counter into the posts table👏. However, as requested by the previous reviewer, please also include a comments counter in the posts table. Because we will be adding comments to a post, I believe it would be a good idea to have a comments counter to help us track the number of comments we receive. 👌😊